docs/index.md

```diff
@@ -14,7 +14,7 @@ You can do this on your own if you'd like, or you can...
 * ```cd victory-tutorial```
 * Replace the existing code in the ```client.js``` file with:
 
-```
+```js
```

Comment:
Yes. I tried `jsx`, and there wasn't any syntax highlighting for some unknown reason, but `js` does have syntax highlighting.